src/oc/web/actions/activity.cljs

```diff
@@ -300,10 +299,15 @@
   (dis/dispatch! [:activity-remove-attachment dispatch-input-key attachment-data]))
 
 (defn get-entry [entry-data]
-  (api/get-entry entry-data
-    (fn [{:keys [status success body]}]
-      (if success
-        (dis/dispatch! [:entry (:uuid entry-data) (clj->js body)])))))
+  (let [is-all-posts (or (:from-all-posts @router/path)
+                         (= (router/current-board-slug) "all-posts"))
+        board-key (if is-all-posts
+                   (dis/all-posts-key (router/current-org-slug))
+                   (dis/board-data-key (router/current-org-slug) (router/current-board-slug)))]
```

Comment:
@thepug you can avoid all this `let` using `dispatcher/current-board-key` that does the same as you do here

Reply:
Yeah, the old cut - n - paste from old code.  Changing!

Reply:
pushed the changes